client/src/main.rs

```diff
@@ -28,7 +29,8 @@ fn main() -> color_eyre::Result<()> {
             initialize_logging()?;
             install_hooks()?;
             term::enable()?;
-            let state = mid::init("http://localhost:8080").await?;
+            let settings = Settings::parse_toml("./server/Server.toml").unwrap();
```

Comment:
will this only change on compilation? if i run the application from the commandline will it look for this specific directory?